tests/system/src/test/java/test/javafx/scene/web/MathMLRenderTest.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (c) 2017, 2018, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.scene.Scene;
+import javafx.scene.layout.StackPane;
+import javafx.scene.web.HTMLEditor;
+import javafx.scene.web.WebEngine;
+import javafx.scene.web.WebView;
+import javafx.stage.Stage;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.Assert;
+
+import static junit.framework.TestCase.fail;
+
+import static test.util.Util.TIMEOUT;
+
+public class MathMLRenderTest {
```

Comment:
Consider writing a headless test under modules/javafx.web/src/test/java/test/javafx/scene/web/.

Reply:
You mean, I have to remove these lines ?
```
            + "   <head>"
            + "      <meta charset=\"UTF-8\">"
            + "      <title>OpenJFX and MathML</title>"
            + "   </head>"
```

Reply:
I really don't  understand your request ?
This test works with all others tests when I use this command :
`gradle -PFULL_TEST=true :systemTests:test`

Reply:
Sorry, I'm really sorry,
I misread the message, but the place you suggest was the first place where I had put the test.
And Kevin said that I had to go into tests/systems ...
I'm a bit confuse about what I have to do ?

![kevin_suggestion](https://user-images.githubusercontent.com/19194678/41795672-da44987a-7663-11e8-8c44-89a34fdc3c47.png)

Reply:
# HeadLess Test

Under modules/javafx.web/src/test/java/test/javafx/scene/web/

- Round 1 : Ok it works with patched WebCore

![screenshot_20180623_005000](https://user-images.githubusercontent.com/19194678/41802277-f8253856-767f-11e8-9eb1-c1c3417cb3f3.png)